extensions/wikia/Search/classes/QueryService/Select/Dismax/AbstractDismax.php

```diff
@@ -3,107 +3,123 @@
 	}
 
 	/**
 	 * Registers our query as an extended dismax query.
+	 *
 	 * @param \Solarium_Query_Select $select
-	 * @return AbstractSelect
+	 *
+	 * @return AbstractDismax
 	 */
 	protected function registerDismax( Solarium_Query_Select $select ) {
-		
+
 		$config = $this->getConfig();
 		$queryFieldsString = $this->getQueryFieldsString();
-		$dismax = $select->getDismax()
-		                 ->setQueryFields( $queryFieldsString )
-		                 ->setQueryParser( 'edismax' )
-		;
-		
+		$dismax = $select->getDisMax()->setQueryFields( $queryFieldsString )->setQueryParser( 'edismax' );
+
 		if ( $this->service->isOnDbCluster() ) {
-			$dismax->setPhraseFields ( $queryFieldsString )
-			       ->setBoostQuery   ( $this->getBoostQueryString() )
-			       ->setMinimumMatch ( $config->getMinimumMatch() )
-			       ->setPhraseSlop   ( 3 )
-				   ->setTie          ( $config->getTie() )
-			;
-			if ( (! $config->getSkipBoostFunctions() ) && (! empty( $this->boostFunctions ) ) ) {
-				$dismax->setBoostFunctions( implode(' ', $this->boostFunctions ) );
+			$dismax->setPhraseFields( $queryFieldsString )
+				->setBoostQuery( $this->getBoostQueryString() )
+				->setMinimumMatch( $config->getMinimumMatch() )
+				->setPhraseSlop( 3 )
+				->setTie( $config->getTie() );
+			if ( ( !$config->getSkipBoostFunctions() ) && ( !empty( $this->boostFunctions ) ) ) {
+				$dismax->setBoostFunctions( implode( ' ', $this->boostFunctions ) );
 			}
 		}
+
 		return $this;
 	}
 
 	/**
 	 * Dismax-specific implementation -- registers the dismax component for child components.
+	 *
 	 * @param \Solarium_Query_Select $select
-	 * @return \Wikia\Search\QueryService\Select\Dismax\AbstractDismax
+	 *
+	 * @return AbstractDismax
 	 */
 	protected function registerComponents( Solarium_Query_Select $select ) {
-		return $this->registerDismax( $select )
-		            ->registerNonDismaxComponents( $select );
+		return $this->registerDismax( $select )->registerNonDismaxComponents( $select );
 	}
 
 	/**
 	 * This is a hook for child components to provide their own specific component registry.
+	 *
 	 * @param \Solarium_Query_Select $select
+	 *
 	 * @internal param \Solarium_Query_Select $query
-	 * @return \Wikia\Search\QueryService\Select\Dismax\AbstractDismax
+	 * @return AbstractDismax
 	 */
 	protected function registerNonDismaxComponents( Solarium_Query_Select $select ) {
 		return $this;
 	}
-	
+
 
 	/**
-	 * As an edismax query, gives the required query in the first clause of the conjunction, and then the parseable query stuff in the second clause.
+	 * As an edismax query, gives the required query in the first clause of the conjunction, and then the parseable
+	 * query stuff in the second clause.
+	 *
 	 * @return string
 	 */
 	protected function getQuery() {
 		$queryClauses = $this->getQueryClausesString();
 		if ( substr_count( $queryClauses, " " ) > 0 ) {
 			$queryClauses = "({$queryClauses})"; // this is just a lucene query syntax nuance
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)